StyleCop.Analyzers/StyleCop.Analyzers.Test/Settings/SettingsFileCodeFixProviderUnitTests.cs

```diff
@@ -64,6 +65,21 @@ namespace NamespaceName
         public async Task TestSettingsFileAlreadyExistsAsync()
         {
             this.createSettingsFile = true;
+            this.settingsFileName = SettingsHelper.SettingsFileName;
+
+            var offeredFixes = await this.GetOfferedCSharpFixesAsync(TestCode).ConfigureAwait(false);
+            Assert.Empty(offeredFixes);
+        }
+
+        /// <summary>
+        /// Verifies that a code fix will not be offered if the settings file is already present.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        public async Task TestDotPrefixedSettingsFileAlreadyExistsAsync()
+        {
+            this.createSettingsFile = true;
+            this.settingsFileName = $".{SettingsHelper.SettingsFileName}";
```

Comment:
❗️ Define a second constant `AltSettingsFileName`:

```csharp
public const string AltSettingsFileName = ".stylecop.json";
```

Reply:
I've added the AltSettingsFileName field to the SettingsHelper class. The implementation of IsStyleCopSettingsFile is also updated to use the new field.